pkg/arvo/sys/vane/gall.hoon

```diff
@@ -1791,12 +1791,16 @@
       ^+  [fex ap-core]
       ?~  caz  [(flop fex) ap-core]
       ?-  i.caz
-        [%pass * %grow *]  $(caz t.caz, ap-core (ap-grow +.q.i.caz))
-        [%pass * %tomb *]  $(caz t.caz, ap-core (ap-tomb +.q.i.caz))
-        [%pass * %cull *]  $(caz t.caz, ap-core (ap-cull +.q.i.caz))
         [%pass * ?(%agent %arvo %pyre) *]  $(caz t.caz, fex [i.caz fex])
         [%give *]  $(caz t.caz, fex [i.caz fex])
         [%slip *]  !!
+        ::
+          [%pass * %grow *]
+        $(caz t.caz, ap-core (ap-grow ['1' +<.q.i.caz] +>.q.i.caz))
+          [%pass * %tomb *]
+        $(caz t.caz, ap-core (ap-tomb +<.q.i.caz ['1' +>.q.i.caz]))
+          [%pass * %cull *]
+        $(caz t.caz, ap-core (ap-cull +<.q.i.caz ['1' +>.q.i.caz]))
```

Comment:
I wasn't picturing that we would change the paths themselves -- I expected we'd just add an explicit version check in handler for the `%x`, `%w`, `%z`, and `%t` cares.